projects/packages/forms/src/contact-form/class-contact-form-plugin.php

```diff
@@ -1627,10 +1645,14 @@ public function get_export_data_for_posts( $post_ids ) {
 			 * If it is not - add an empty string, which is just a placeholder in the CSV.
 			 */
 			foreach ( $field_names as $single_field_name ) {
+				$renamed_field = isset( $well_known_column_names[ $single_field_name ] )
+					? $well_known_column_names[ $single_field_name ]
+					: $single_field_name;
+
 				/**
 				 * Remove the numeral prefix 1_, 2_, etc, only for export results
 				 */
-				$renamed_field = preg_replace( '/^(\d{1,2}_)/', '', $single_field_name );
+				$renamed_field = preg_replace( '/^(-?\d{1,2}_)/', '', $renamed_field );
```

Comment:
I wonder if we should expand the comment a bit, for our future selves? to explain why we need to care about a possible `-` here?

Reply:
Added some more explanation on how/why prefixes could be negative values and a reference to the sorting function. Hope that's enough